telegram/bot.py

```diff
@@ -2408,6 +2416,45 @@ def ban_chat_member(
 
         return result  # type: ignore[return-value]
 
+    @log
+    def ban_chat_sender_chat(
+        self,
+        chat_id: Union[str, int],
+        sender_chat_id: int,
+        timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: JSONDict = None,
+    ) -> bool:
+        """
+        Use this method to ban a channel chat in a supergroup or a channel. Until the chat is
+        unbanned, the owner of the banned chat won't be able to send messages on behalf of *any of
```

Comment:
```suggestion
        unbanned, the owner of the banned chat won't be able to send messages on behalf of *any of
```
`**any of their channels**`

Reply:
yep changed this locally didn't commit yet